java/arcs/sdk/jvm/ParticleImpl.kt

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.sdk
+
+/** Implementation of [Particle] for the JVM. */
+abstract class ParticleImpl : Particle
```

Comment:
It's not really an Impl... maybe "BaseParticle"?

Reply:
Done